### DISCUSSION TITLE
Got 404 error on files with `.md` extension

### DISCUSSION THREAD

**Discussion by PRO-2684** [community/community#33458] [Other Feature Feedback, Questions, & Ideas] [answered]
Here's my [blog link](https://pro-2684.github.io/) and [Github repository](https://github.com/PRO-2684/PRO-2684.github.io).
I've uploaded the markdown files under `/notes` directory, but when I tried to access them (i.e. [index.md](https://pro-2684.github.io/notes/index.md)), I got 404. Other files can be accessed successfully.

**Comment by MacaylaMarvelous81** [comment, marked as answer]
You can access them without .md extension.
Example:
(X) https://pro-2684.github.io/notes/index.md
(O) https://pro-2684.github.io/notes/index

**Reply by PRO-2684** [reply]
That's neat, but I'd want to access the source code of my markdown files.

**Reply by MacaylaMarvelous81** [reply]
You can put a `.nojekyll` empty file into the `notes` folder to exclude it from jekyll, and you will be able to access both (rendered and source).

**Reply by PRO-2684** [reply]
Placing it under the root directory helps. Thank you!
